### PR TITLE
Vendor in latest code for storage,image, buildah

### PR DESCRIPTION
**Pull request by rhatdan**
vendor in containers/storage
vendor in containers/image
vendor in projectatomic/buildah

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: libpod/storage.go [hunk 1]
```diff
@@ -231,21 +231,21 @@ func (r *storageService) MountContainerImage(idOrName string) (string, error) {
 	return mountPoint, nil
 }
 
-func (r *storageService) UnmountContainerImage(idOrName string) error {
+func (r *storageService) UnmountContainerImage(idOrName string) (bool, error) {
```
Comment (mheon) [inline libpod/storage.go]: Might need to change this for the (int, error) return change

**Comment by rhatdan** [comment]
@baude @giuseppe @umohnani8 PTAL

File: libpod/storage.go (hunk 1 above)
Comment (mheon) [reply libpod/storage.go]: Ah, fair enough - works for me.

**Comment by mheon** [comment]
LGTM

**Comment by mheon** [comment]
bot, retest this please

**Comment by umohnani8** [comment]
LGTM

**Comment by mheon** [comment]
bot, retest this please

**Comment by mheon** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**